psl-clj/template/project.clj

```diff
@@ -0,0 +1,6 @@
+(defproject psl-clj/lein-template "2.1.0-SNAPSHOT"
```

Comment:
So is this template some sort of example?
If so, lets move it from here into the psl-examples repo.